### ISSUE TITLE
no method 'unique'

### ISSUE THREAD

**Issue by JackvdKlooster** [gustavohenke/grunt-jquery-builder#2]
Hi,

I'm using your jquery builder for a few weeks now and we are very happy with that. Since a few days i'm getting this warning in my terminal:

```
Running "jquery" (jquery) task
Warning: Object  has no method 'unique' Use --force to continue."
```

Am i doing something wrong or is the builder broken?

I wrote my taskt like:

```
jquery: {
    dist: {
        options: {
            prefix: "000_jquery-",
        },
        output: "js/libraries",
        versions: ["1.10.0"]
    }
}
```

**Comment by gustavohenke** [comment]
Could you please run this command for me and show its the output?

```
npm ls --depth=0
```

**Comment by gustavohenke** [comment]
Don't worry about my previous comment.

The fix landed in v0.2.1. All you need to do is update grunt-jquery-builder!
